autolens/pipeline/phase/abstract/result.py

```diff
@@ -31,6 +33,62 @@ def __init__(
     def most_likely_tracer(self):
         return self.analysis.tracer_for_instance(instance=self.instance)
 
+    @property
+    def source_plane_light_profile_centres(self):
+        if self.most_likely_tracer.source_plane.has_light_profile:
+            return self.most_likely_tracer.light_profile_centres_of_planes[-1]
+        else:
+            return []
+
+    @property
+    def source_plane_inversion_centres(self):
+        if self.most_likely_tracer.source_plane.has_pixelization:
+            return [
+                self.most_likely_fit.inversion.brightest_reconstruction_pixel_centre
+            ]
+        else:
+            return []
+
+    @property
+    def source_plane_centres(self):
+        centres = (
+            self.source_plane_light_profile_centres
+            + self.source_plane_inversion_centres
```

Comment:
Would also be nice to get some type annotation on these methods.

This one would be:
```python
def source_plane_centres(self) -> grid.Coordinates:
```